wherehows-web/app/router.ts

```diff
@@ -42,7 +43,7 @@ const AppRouter = EmberRouter.extend({
        * @link https://developer.piwik.org/guides/tracking-javascript-guide#internal-search-tracking
        */
       if (title.includes('search') && searchInfo) {
-        const { keywords, category = 'datasets', page = 1 } = searchInfo.context;
+        const { keywords, category = 'datasets', page = 1 } = <{ [key: string]: any }>searchInfo.context;
```

Comment:
you can use `Record<string, any>` here, instead of <{ [key: string]: any }>

Reply:
Cool